test/fastly/client_test.rb

```diff
@@ -139,4 +148,4 @@
 
     end
   end
-end
+end
```

Comment:
This change was likely accidental. Let's revert this change too since you'll be changing this file.